### DISCUSSION TITLE
Capacitor support

### DISCUSSION THREAD

**Discussion by chris-si** [electric-sql/electric#1424] [Feature requests]
Currently there doesn't seem to be an official driver/plugin to use electric-sql with capacitor. The mobile docs only mention Expo and React Native atm. It would be great if capacitor apps would also be officially supported.

**Comment by KyleAMathews** [comment]
👋 we've been working the last month on a rebuild of the Electric server over at a temporary repo https://github.com/electric-sql/electric-next/

You can read more about why we made the decision at https://next.electric-sql.com/about

We're really excited about all the new possibilities the new server brings and we hope you'll check it out soon and give us your feedback.

We're now moving the temporary repo back here. As part of that migration we're closing all the old issues, PRs, and discussions. We really appreciate you taking the time to start this discussion! If it's still relevant in the new system, please reopen.
